src/components/MsvcError/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { h, FunctionComponent } from 'preact'
+import classNames from 'classnames'
+import { Button } from '@onfido/castor-react'
+
+import { useLocales } from '~locales'
+import theme from 'components/Theme/style.scss'
+import ScreenLayout from '../Theme/ScreenLayout'
+import style from './style.scss'
+
+import type { StepComponentBaseProps } from '~types/routers'
+import PageTitle from 'components/PageTitle'
+
+type MsvcErrorType = 'GENERAL' | 'RESTART' | 'UNVERIFIABLE'
+
+const titleKey = (key: MsvcErrorType): string => {
```

Comment:
Is this function doing anything but changing the key to lower case? couldn't we use toLowerCase()

Reply:
That is what it does, but unfortunately, `toLowerCase()` didn't work for me. I ended getting back empty strings. I used our `lowerCase` function built-in to the SDK, which improves on a few dangers with the default `toLowerCase()` function. I suspect that the MsvcErrorType is converted to a non-string under the hood when compiled, thus making them not actually strings.